schemas/oic.r.coapcloudconf-schema.json

```diff
@@ -1,43 +1,38 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
-  "id": "https://www.openconnectivity.org/ocf-apis/core/schemas/oic.r.coapcloudconf-schema.json#",
+  "id": "https://www.openconnectivity.org/ocf-apis/core-extensions/schemas/oic.r.coapcloudconf-schema.json#",
   "definitions": {
     "oic.r.coapcloudconf": {
       "type": "object",
       "properties": {
         "apn": {
           "type": "string",
-          "description": "Authentication provider's name issuing the auth code",
+          "description": "The Authorisation Provider through which an Access Token was obtained.",
           "pattern": "^.*$"
         },
         "cis": {
           "type": "string",
-          "description": "Indicates URL for CoAP native cloud interface server to be registered",
+          "description": "URL of OCF Cloud",
           "format": "uri"
         },
         "at": {
           "type": "string",
-          "description": "Indicates access token which is given in return of auth code issued by an appropriate account server",
-          "pattern": "^.*$"
-        },
-        "sid": {
```

Comment:
Need to add "sid" back into the schemas.

Reply:
Yes, coming soon in other open PR #4

Reply:
Updated in PR #5 (Now Merged).